pkg/controller/common/annotation/constants.go

```diff
@@ -25,4 +25,6 @@ const (
 
 	ElasticsearchConfigAndSecretMountsHashAnnotation = "policy.k8s.elastic.co/elasticsearch-config-mounts-hash" //nolint:gosec
 	SourceSecretAnnotationName                       = "policy.k8s.elastic.co/source-secret-name"               //nolint:gosec
+
+	SoftOwnerRefsAnnotation = "eck.k8s.elastic.co/owner-refs"
```

Comment:
Should we keep with close to the existing labels used for soft owner refs?

Reply:
moved this closer to the existing labels here dd18a2d5c5167fc0369c9b1915a26d147cc88874